rllib/core/rl_module/torch/torch_rl_module.py

```diff
@@ -93,3 +97,12 @@ def _module_metadata(self, *args, **kwargs):
     @override(RLModule)
     def unwrapped(self) -> "RLModule":
         return self.module
+
+
+class TorchDDPRLModuleWithTargetNetworksInterface(
```

Comment:
note: this feels awkward, but is probably the best thing to do in the current situation

Reply:
Agree. I wasn't thinking much of a cleaner solution. Just tried to make this work for now. I'll add a TODO.
I don't like it either that we have to create these awkward wrappers only such that DDP will work, b/c yes, it means we'll have to create a special wrapper for each sub-class of `RLModule` that we'll come up with in the future. :(